features/leeftijd_bepaling.feature

```diff
@@ -35,11 +35,27 @@ Scenario: Alleen jaar van geboorte datum is bekend
 	Dan is attribuut leeftijd niet aanwezig
 
 Scenario: Persoon is overleden
-	Gegeven een ingeschreven persoon met geboortedatum 26 mei 1953
-	En de ingeschreven persoon is overleden
-	Als de ingeschreven persoon op <raadpleeg datum> wordt geraadpleegd
+	Gegeven de persoon met burgerservicenummer "999993197" is overleden op 28-04-2019
+	Als de ingeschreven persoon met burgerservicenummer 999994268 wordt geraadpleegd met fields=leeftijd
 	Dan is attribuut leeftijd niet aanwezig
 
+Scenario: leeftijd wordt wel geleverd bij een overleden kind (omdat alleen gegevens van de persoonslijst van de gevraagde persoon worden gebruikt)
+	Gegeven de persoon met burgerservicenummer "999991280" heeft een kind met burgerservicenummer "999993197" en voornamen "Henk-Pieter"
+	En de persoon met burgerservicenummer "999993197" is overleden op 28-04-2019
+	Als de ingeschreven persoon met burgerservicenummer 999994268 wordt geraadpleegd met fields=kinderen
+	Dan heeft het kind met burgerservicenummer "999993197" naam.voornamen "Henk-Pieter" een attribuut "leeftijd" met een waarde
+
+Scenario: bij een levenloos geboren kind wordt geen leeftijd geleverd
+	Gegeven de persoon met burgerservicenummer "999994268" heeft de volgende gegevens voor kinderen:
+		| categorie | voornamen (02.10) | geboortedatum (03.10) | Registratie betrekking (89.10) |
+		| 9         | Truus             | 19600912              |                                |
+		| 9         | Tinie             | 19600912              | L                              |
+	En de gebruiker heeft een autorisatie met rubriek 35.95.14 Bijzondere betrekking verstrekken met de waarde “1”
+	Als de ingeschreven persoon met burgerservicenummer 999994268 wordt geraadpleegd met fields=kinderen
+	Dan bevat het antwoord 2 kinderen
+	En heeft het kind met naam.voornamen "Truus" een attribuut "leeftijd" met een waarde
+	En heeft het kind met naam.voornamen "Tinie" GEEN attribuut "leeftijd"
```

Comment:
Ziet er goed uit, nog even voor de zekerheid:

Als de gebruiker geen autorisatie heeft op rubriek Registratie betrekking (89.10) dan wordt de leeftijd wel meegegeven? Ofwel, ook hier geldt dat alleen wat is geautoriseerd wordt gebruikt voor het samenstellen van gegevens?

Reply:
@hvdijk-p5 je bedoelt dat de gebruiker wel geautoriseerd is om levenloos geboren kinderen te krijgen (35.95.14=1), maar niet geautoriseerd is voor 09.89.10? Dat is hopelijk een theoretisch geval, maar in die situatie geldt inderdaad nog steeds "alleen wat is geautoriseerd wordt gebruikt voor het samenstellen van gegevens"

Reply:
@hvdijk-p5 en @fsamwel Als een afnemer niet is geautoriseerd voor "registratie betrekking" dan krijgt hij ook geen leeftijd. Het antwoord is nee, alle gegevens worden gebruikt voor het samenstellen van de gegevens. Er geldt alleen een filter bij het verstrekken van de gegevens. Misschien moeten we even vragen hoe het zit met deze autorisatie. Ik neem aan dat 
1. gebruikers die levenloze kinderen van persoon mogen raadplegen automatisch geautoriseerd zijn voor registratie betrekking (anders kun je dit niet goed interpreteren).
2. gebruikers die geen levenloze kinderen van persoon mogen raadplegen geen enkel gegeven van het levenloze kind mogen raadplegen.